SpatialGDK/Source/SpatialGDK/Private/Interop/ActorSystem.cpp

```diff
@@ -299,6 +299,8 @@ void ActorSystem::Advance()
 		ProcessAdds(SubView);
 	}
 
+	SendRepNotifies();
```

Comment:
Nit - name is a bit weird as nothing is being sent. I'd possibly call it Invoke instead. But this doesn't really matter.

Reply:
Changed!